sky/packages/sky/lib/src/fn3/scrollable.dart

```diff
@@ -126,7 +126,10 @@ abstract class ScrollableState<T extends Scrollable> extends State<T> {
   }
 
   bool _scrollOffsetIsInBounds(double offset) {
-    return offset >= scrollBehavior.minScrollOffset && offset < scrollBehavior.maxScrollOffset;
+    if (scrollBehavior is! ExtentScrollBehavior)
+      return false;
+    ExtentScrollBehavior behavior = scrollBehavior as ExtentScrollBehavior;
```

Comment:
don't need the as (and having it is more expensive in release mode). Just do the assignment.